recipe/meta.yaml

```diff
@@ -56,6 +56,9 @@ requirements:
     - git
 
 test:
+  # add requires to ensure that all variants of cuda_compiler_version are tested
+  requires:                                    # [linux and not (aarch64 or ppc64le) and cuda_compiler_version != "None"]
+    - cudatoolkit {{ cuda_compiler_version }}  # [linux and not (aarch64 or ppc64le) and cuda_compiler_version != "None"]
```

Comment:
Can you eliminate `not (aarch64 or ppc64le)`? Suggestion:
```
    - cudatoolkit {{ cuda_compiler_version }}  # [cuda_compiler_version not in (undefined, "None")]
```

Reply:
I've had problems with `undefined` before, but let's see if it works.

Reply:
(and it's not like what we do now is set in stone; if the aarches gain cpu support, it would be easy to make an adaptation here.)

Reply:
(sure, the suggestion for just to minimize any changes to be made in future).